config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   root "home#index"
-  resources :home, only: [:index, :edit, :update]
+  resources :home, only: [:index, :edit, :update, :create]
```

Comment:
When I added to form to the home page, my server threw a route error, because it was looking for the create route on the home page.  Want me to just point this form at the Companies Controller?